apps/web/src/app/mypage/page.module.css

```
@@ -0,0 +1,35 @@
+  min-height: 100dvh;
+  background-color: #242426; /* gray.1000 */
+}
+
+.srOnly {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.3.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)